src/js/selection.js

```diff
@@ -1,18 +1,15 @@
-/*global Util */
-var Selection;
-
 (function () {
     'use strict';
 
     function filterOnlyParentElements(node) {
-        if (Util.isBlockContainer(node)) {
+        if (MediumEditor.util.isBlockContainer(node)) {
             return NodeFilter.FILTER_ACCEPT;
         } else {
             return NodeFilter.FILTER_SKIP;
         }
     }
 
-    Selection = {
+    var Selection = {
```

Comment:
The thought here was to keep the references within the file short (`Selection` instead of `MediumEditor.selection`) in both `Selection` and `Util`.  Just define a local variable for references within the file, and at the end write it to the global variable.  Kind of like how you do things with `define()` or `exports`.

Do you think there's more value in using the global reference?